follow_test.go

```diff
@@ -871,7 +871,7 @@ func Test_following(t *testing.T) {
 						UserName: "TwitterData",
 					},
 				},
-				Meta: gotwtr.FollowsMeta{
```

Comment:
Why did you change ?

Reply:
In other test codes, meta struct was passed by address, but only here it was passed by value, so I changed it.